engines/agi/inv.cpp

```diff
@@ -187,10 +198,16 @@ void InventoryMgr::keyPress(uint16 newKey) {
 		changeActiveItem(+2);
 		break;
 	case AGI_KEY_LEFT:
-		changeActiveItem(-1);
+		if (!_vm->isLanguageRTL())
+			changeActiveItem(-1);
+		else
+			changeActiveItem(+1);
```

Comment:
I prefer to be conservative, and do the minimum changes, to make it explicit that the original non RTL behaviour hasn't changed at all.
Therefore, I don't want to add helper functions such as these, and I prefer to not use the ternary, since IMO with an `if` it's more clear that nothing changed in the regular case.